RpcWallet/RpcWallet.cs

```diff
@@ -82,17 +77,15 @@ public JObject OnProcess(HttpContext context, string method, JArray _params)
                             from = _params[0].AsString().ToScriptHash();
                             to_start = 1;
                         }
-                        JArray to = (JArray)_params[to_start + 0];
-                        long fee = _params.Count >= to_start + 2 ? long.Parse(_params[to_start + 1].AsString()) : 0;
```

Comment:
We should be able to attach ScopedSignatures, but maybe in other pull request

Reply:
Let's do in another PR, scoped was not merged, right?